src/compiler/utilities.ts

```diff
@@ -385,6 +385,12 @@ namespace ts {
         return (identifier.length >= 2 && identifier.charCodeAt(0) === CharacterCodes._ && identifier.charCodeAt(1) === CharacterCodes._ ? "_" + identifier : identifier) as __String;
     }
 
+    export function isEscapedNameOfWellKnownSymbol(escapedName: __String) {

```

Comment:
With #15473, this won't have anything to do with well-known symbols.

Reply:
But for now, it's perfectly accurate.

Reply:
can we move this to services/utilities instead?

Reply:
I can move it. I'll rename it later.

Reply:
Actually, I'll just inline it into `getCompletionEntryDisplayNameForSymbol` since we do similar tests in there.